src/model/test/SubSurface_GTest.cpp

```diff
@@ -1280,3 +1281,244 @@ TEST_F(ModelFixture, SubSurface_isXXXAutocalculated_4399) {
   subSurface.autocalculateNumberofVertices();
   EXPECT_TRUE(subSurface.isNumberofVerticesAutocalculated());
 }
+
+// https://github.com/NREL/OpenStudio/issues/4361
+TEST_F(ModelFixture, Issue_4361) {
+  Model model;
+
+  ThermalZone thermalZone(model);
+
+  Space space(model);
+  space.setThermalZone(thermalZone);
+
+  // Create a surface and subsurface
+  // get the surface net and gross area & subsurface gross area
+  // surface gross area == surface net area + subsurface gross area
+  // That is before the subsurface is assigned a frame and divider
+
+  std::vector<Point3d> vertices{
+    {0, 0, 2},
+    {0, 0, 0},
+    {2, 0, 0},
+    {2, 0, 2},
+  };
+  Surface surface(vertices, model);
+  surface.setSpace(space);
+  surface.setSurfaceType("Wall");
+
+  vertices.clear();
+  vertices = {
+    {0.5, 0, 1.5},
+    {0.5, 0, 0.5},
+    {1.5, 0, 0.5},
+    {1.5, 0, 1.5},
+  };
+
+  SubSurface subSurface(vertices, model);
+  subSurface.setSurface(surface);
+  subSurface.setSubSurfaceType("FixedWindow");
+
+  double surfaceGrossArea = surface.grossArea();
+  double surfaceNetArea = surface.netArea();
+  double subSurfaceGrossArea = subSurface.grossArea();
+  double subSurfaceTotalArea = subSurface.roughOpeningArea();
+
+  EXPECT_EQ(surfaceGrossArea, surfaceNetArea + subSurfaceGrossArea);
+  double windowWallRatio = surface.windowToWallRatio();
+  EXPECT_EQ(windowWallRatio, 0.25);
+
+  // Then assign a frame and divider to the subsurface
+  // then do the same thing with the areas
+
+  WindowPropertyFrameAndDivider frame(model);
+  frame.setFrameWidth(0.030);
+
+  subSurface.setWindowPropertyFrameAndDivider(frame);
+
+  double frameArea = subSurface.frameArea();
+  EXPECT_NEAR(frameArea, 0.1236, 0.01);
+  double divArea = subSurface.dividerArea();
+  EXPECT_DOUBLE_EQ(divArea, 0);
```

Comment:
I'd like to see a test for this (that isn't 0)

Reply:
Ok

Reply:
Done